src/main/java/org/apache/commons/imaging/formats/tiff/TiffField.java

```diff
@@ -288,6 +296,55 @@ public int getIntValueOrArraySum() throws ImagingException {
         // return -1;
     }
 
+    public long[] getLongArrayValue() throws ImagingException {
```

Comment:
We should provide Javadocs on all new public and protected methods IMO.

Reply:
done